tests/should_succeed/lang/unsized_quant.rs

```diff
@@ -1,14 +1,12 @@
-#![allow(incomplete_features)]
-#![feature(unsized_locals)]
-
+// TODO: keep this test?
```

Comment:
Rust removed the feature `unsized_locals`. If we can't find a workaround then this test is kinda pointless.